### PR TITLE
Fix/et 1134 email from displays encoded html entities

### PR DESCRIPTION
**Pull request by redscar**
### 🎫 Ticket

[ET-1134] 

### 🗒️ Description

Single quotes were being escaped. Gmail did not like this. I removed the escaping that was occurring on the persons name

Bug Fix -

Single line change for TC sending emails for Paypal.

### 🎥 Artifacts 

https://user-images.githubusercontent.com/12241059/122456218-4e81f980-cf7b-11eb-86c5-d9eb7d700884.mp4

### ✔️ Checklist
- [x] I've included a changelog entry both in readme.txt and changelog.txt fi>
- [x] My code is tested. <!-- Check that tests are passing and DO NOT merge i>
- [x] My code has [proper inline documentation](https://the-events-calendar.g>

[ET-1134]: https://theeventscalendar.atlassian.net/browse/ET-1134

### CODE REVIEW

**Comment by juanfra** [comment]
@redscar looks like we need to update the base branch for this PR

**Comment by redscar** [comment]
@juanfra I just updated the branch to carnotaurus

File: changelog.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 == Changelog ==
-
+* Fix - Prevent Tribe Commerce 'Confirmation email sender name' from displaying improperly when a single quote is added. [ET-1134]
```
Comment (juanfra) [inline changelog.txt:2]: Let's use double quotes and move it to the right version (Below [5.1.5])
```suggestion
* Fix - Prevent Tribe Commerce "Confirmation email sender name" from displaying improperly when a single quote is added. [ET-1134]
```

File: readme.txt [hunk 2]
```diff
@@ -181,7 +181,7 @@ Check out our extensive [knowledgebase](https://evnt.is/18wm) for articles on us
 == Changelog ==
 
 = [5.1.6] TBD =
-
+* Fix - Prevent Tribe Commerce 'Confirmation email sender name' from displaying improperly when a single quote is added. [ET-1134]
```
Comment (juanfra) [inline readme.txt:184]:
```suggestion
* Fix - Prevent Tribe Commerce "Confirmation email sender name" from displaying improperly when a single quote is added. [ET-1134]
```
Let's use double quotes.

**Review by juanfra — CHANGES_REQUESTED** [review]

**Review by juanfra — APPROVED** [review]

**Pull request merged**